File: packages/expo-analytics-amplitude/android/src/main/java/expo/modules/analytics/amplitude/AmplitudePackage.java
package expo.modules.analytics.amplitude;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.core.BasePackage;
import expo.core.ExportedModule;
import expo.core.ViewManager;

public class AmplitudePackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new AmplitudeModule(context));
  }

  @Override
  public List<ViewManager> createViewManagers(Context context) {
    return Collections.singletonList((ViewManager) new AmplitudeViewManager());
  }
}

File: packages/expo-analytics-amplitude/android/src/main/java/expo/modules/analytics/amplitude/AmplitudePackage.kt
package expo.modules.analytics.amplitude

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class AmplitudePackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(AmplitudeModule(context))
  }
}
